src/prisma/schema.prisma

```diff
@@ -3507,3 +3511,19 @@ model ScoutProjectDeployer {
   @@index([projectId])
   @@index([address])
 }
+
+enum OnchainBuilderEventType {
+  registered
+  banned
+  unbanned
+}
+
+model OnchainBuilderEvent {
```

Comment:
This is not an 'onchain version' of builder event, can we use something besides builder event? Like "BuilderStatusEvent"